airbyte-ci/connectors/pipelines/pipelines/cli/airbyte_ci.py

```diff
@@ -207,13 +235,22 @@ async def airbyte_ci(
     ci_git_user: str,
     ci_github_access_token: str,
     ci_report_bucket_name: str,
+    ci_artifact_bucket_name: str,
     ci_gcs_credentials: str,
     ci_job_key: str,
     s3_build_cache_access_key_id: str,
     s3_build_cache_secret_key: str,
     show_dagger_logs: bool,
 ):  # noqa D103
     ctx.ensure_object(dict)
+
+
+    if enable_dagger_run and not is_current_process_wrapped_by_dagger_run():
+        main_logger.info("Re-Running airbyte-ci with dagger run.")
```

Comment:
I think this log might be a bit confusing to users when it does this each time by default 🤔